python/dgl/convert.py

```diff
@@ -1653,11 +1752,15 @@ def to_networkx(g, node_attrs=None, edge_attrs=None):
     Parameters
     ----------
     g : DGLGraph
-        A homogeneous graph.
+        A homogeneous or heterogeneous graph.
     node_attrs : iterable of str, optional
         The node attributes to copy from ``g.ndata``. (Default: None)
     edge_attrs : iterable of str, optional
         The edge attributes to copy from ``g.edata``. (Default: None)
+    ntype_attr : str, optional
+        The node attribute to write the node type to. (Default: "type")
+    etype_attr : str, optional
+        The edge attribute to write the edge type to. (Default: "triples")
```

Comment:
Ditto. Also, will you store the edge type or the canonical edge type?

Reply:
My understanding is that, according to the original conversation, the canonical type (as a triple) is preferred here. I also think this is more flexible, as the edge type name can always be extracted from the canonical type if that's needed.

To summarize, the approach here is to save the node type as the `ntype` attribute on NetworkX nodes and the (canonical) edge type as the `etype` attribute on NetworkX edges. Is this still a good assumption?

Reply:
Yes. You just need to make that point clear in the documentation.